samples/subsys/trusted_storage/src/main.c

```diff
@@ -13,6 +13,42 @@
 #define TEST_STRING_1 "The quick brown fox jumps over the lazy dog"
 #define TEST_STRING_2 "Lorem ipsum dolor sit amet"
 
+
+/* Use settings with littlefs backend */
+#ifdef CONFIG_TRUSTED_STORAGE
+
+#include "settings/settings_file.h"
+#include <zephyr/device.h>
+#include <zephyr/fs/fs.h>
+#include <zephyr/fs/littlefs.h>
+
+#define STORAGE_PARTITION	storage_partition
+#define STORAGE_PARTITION_ID	FIXED_PARTITION_ID(STORAGE_PARTITION)
+
+/* NFFS work area strcut */
```

Comment:
```suggestion
/* NFFS work area struct */
```

Reply:
It stands for Newtron Flash File System and seems to be an FS that Zephyr had before so this is most likely just copy-pasted from somewhere.... So changing to LittelFS